# 5 for arm, 3 for base
float32[8] action
---
geometry_msgs/Pose object_pose
sensor_msgs/JointState joint_state
control_msgs/JointTrajectoryControllerState base_state
nav_msgs/Odometry odom
float32 reward
int32 is_done